### ISSUE TITLE
Java properties vs environment variables

### ISSUE THREAD

**Issue by o-pikozh** [harawata/appdirs#6]
Either I am missing something, or you read XDG variables in the wrong place.

`XDG_DATA_HOME`, `XDG_RUNTIME_DIR`, etc are environment variables. But you're trying to read them using `System.getProperty`.

**Comment by harawata** [comment]
Oh, I've missed this issue for some reason.
I'll look into it.

**Comment by harawata** [comment]
Hi @o-pikozh ,

You are right!
I've fixed it locally, but just to be sure, could you verify the fix with the latest [1.0.3-SNAPSHOT](https://oss.sonatype.org/content/repositories/snapshots/net/harawata/appdirs/) before I release the final version?

If you use Maven, add the snapshot repository to the pom.xml...

```xml
<project>
...
  <repositories>
    <repository>
      <id>sonatype-snapshot</id>
      <name>Sonatype OSS Snapshot Repository</name>
      <url>http://oss.sonatype.org/content/repositories/snapshots</url>
    </repository>
  </repositories>
...
</project>
```
...and specify `1.0.3-SNAPSHOT` as appdirs version.

```xml
<dependency>
  <groupId>net.harawata</groupId>
  <artifactId>appdirs</artifactId>
  <version>1.0.3-SNAPSHOT</version>
</dependency>
```

If you're busy, that's fine. I'll just release it. :)

**Comment by harawata** [comment]
1.0.3 is published.
Thanks for your help!
